app-modules/assistant/resources/views/filament/pages/ai-assistant.blade.php

```diff
@@ -2,207 +2,367 @@
+                                    >
+                                    </textarea>
+                                </div>
+                                <div class="flex items-center justify-between border-t px-3 py-2 dark:border-gray-600">
+                                    <div class="flex items-center space-x-2">
+                                        <button
+                                            class="inline-flex items-center rounded-lg bg-primary-500 px-4 py-2.5 text-center text-xs font-medium text-white hover:bg-primary-700 focus:ring-4 focus:ring-primary-200 dark:focus:ring-primary-900"
+                                            type="submit"
+                                            wire:loading.remove
+                                            x-on:click="$wire.showCurrentResponse = true"
+                                        >
+                                            Post
+                                        </button>
+                                        <svg
+                                            class="-ml-1 mr-3 h-5 w-5 animate-spin text-primary-600"
+                                            wire:loading
+                                            xmlns="http://www.w3.org/2000/svg"
+                                            fill="none"
+                                            viewBox="0 0 24 24"
+                                        >
+                                            <circle
+                                                class="opacity-25"
+                                                cx="12"
+                                                cy="12"
+                                                r="10"
+                                                stroke="currentColor"
+                                                stroke-width="4"
+                                            ></circle>
+                                            <path
+                                                class="opacity-75"
+                                                fill="currentColor"
+                                                d="M4 12a8 8 0 018-8V0C5.373 0 0 5.373 0 12h4zm2 5.291A7.962 7.962 0 014 12H0c0 3.042 1.135 5.824 3 7.938l3-2.647z"
+                                            ></path>
+                                        </svg>
+                                        @error('message')
+                                            <p class="ml-auto text-xs text-red-500">{{ $message }}</p>
+                                        @enderror
+                                    </div>
+                                    <div class="flex space-x-1 pl-0 sm:pl-2">
+                                        @if (!$chat->id && $chat->messages->count() > 0)
+                                            {{ $this->saveChatAction }}
+                                        @endif
```

Comment:
Can we indicate that a chat is not yet saved by having the icon render outlined initially?

Reply:
Well, it won't appear unless the can save it. We have also added a label to indicate it can be saved now.